pkgs/development/interpreters/micropython/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, lib, fetchFromGitHub, pkgs }:
+
+stdenv.mkDerivation rec {
+  pname = "micropython";
+  version = "1.11";
+
+  src = fetchFromGitHub {
+    owner  = "micropython";
+    repo   = "micropython";
+    rev    = "v${version}";
+    sha256 = "16pj6xpiai4y4g8fhn3b824zibgfv62l4ib091pj930q07g109bv";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = with pkgs; [
+    pkgconfig libffi python3 readline
```

Comment:
pkgconfig belongs in `nativeBuildInputs`

Reply:
Fixed